glitch/analysis/security.py

```diff
@@ -1,15 +1,1107 @@
+import os
 import re
 import json
 import configparser
 from urllib.parse import urlparse
-from glitch.analysis.rules import Error, RuleVisitor
+from glitch.analysis.rules import Error, RuleVisitor, SmellChecker
+from nltk.tokenize import WordPunctTokenizer
 
+from glitch.tech import Tech
 from glitch.repr.inter import *
 
 
 class SecurityVisitor(RuleVisitor):
     __URL_REGEX = r"^(http:\/\/www\.|https:\/\/www\.|http:\/\/|https:\/\/)?[a-z0-9]+([_\-\.]{1}[a-z0-9]+)*\.[a-z]{2,5}(:[0-9]{1,5})?(\/.*)?$"
 
+    class TerraformSmellChecker(SmellChecker):
```

Comment:
I think these smell checkers should be on a different package and each of the checkers should have its own file. I think this will make the code easier to read.